code/game/turfs/unsimulated.dm

```diff
@@ -12,34 +12,3 @@
 
 /turf/unsimulated/acid_act(acidpwr, acid_volume, acid_id)
 	return 0
-
-/turf/unsimulated/floor/plating/vox
-	icon_state = "plating"
-	name = "plating"
-	nitrogen = 100
-	oxygen = 0
-
-/turf/unsimulated/floor/plating/snow
-	name = "snow"
-	icon = 'icons/turf/snow.dmi'
-	icon_state = "snow"
-	temperature = T0C
-
-/turf/unsimulated/floor/plating/snow/concrete
```

Comment:
`snow/concrete`

Sorry **WHAT?**

Reply:
![image](https://user-images.githubusercontent.com/11361525/106579178-0e305a80-6549-11eb-9e01-4b22ef57ac41.png)

Reply:
Look at the git history and everything will make sense.

Reply:
Name checks out